yt_dlp/extractor/amcnetworks.py

```diff
@@ -90,30 +114,41 @@ def _real_extract(self, url):
         formats, subtitles = self._extract_theplatform_smil(
             media_url, video_id)
         self._sort_formats(formats)
+
+        thumbnails = []
+        thumbnail_urls = [properties.get('imageDesktop')]
+        if 'thumbnail' in info:
+            thumbnail_urls.append(info.pop('thumbnail'))
+        for thumbnail_url in thumbnail_urls:
+            if not thumbnail_url:
+                continue
+            mobj = re.search(r'(\d+)x(\d+)', thumbnail_url)
+            thumbnails.append({
+                'url': thumbnail_url,
+                'width': int(mobj.group(1)) if mobj else None,
+                'height': int(mobj.group(2)) if mobj else None,
+            })
+
         info.update({
+            'age_limit': parse_age_limit(parse_age_limit(rating)),
```

Comment:
`parse_age_limit` is called twice?

Reply:
That was in the original code. I hadn't paid any attention to it.

Now that I've read parse_age_limit, however, I noticed that `US_RATINGS` and `TV_PARENTAL_GUIDELINES` are case-sensitive although some of the JSON data uses those values lowercase.